libs/gf/configuration-danish-locale/src/lib/danish-locale.integration.spec.ts

```diff
@@ -35,11 +39,12 @@ class DummyPipe implements PipeTransform {
 describe('Danish locale', () => {
   it('configures the DecimalPipe', () => {
     const harness = createPipeHarness({
-      imports: [DanishLocaleModule],
+      imports: [TranslocoModule],
```

Comment:
suggestion: doesn't look like `TranslocoModule` needs to be imported in this file. There's nothing that uses Transloco here.